protobuf/full_snapshot.proto

```diff
@@ -625,3 +629,37 @@ message CustomTypeInformation {
     string type = 2;
   }
 }
+
+message QueryPlanInformation {
+  int32 query_idx = 1;
+  int64 plan_id = 2;
+  string explain_plan = 3;
+  google.protobuf.Timestamp plan_captured_time = 4;
```

Comment:
It might be helpful to track a boolean flag here as well, to remember whether the `plan_type` was `actual` - that way we could consider creating query samples for that in the future on the server side, if someone has enabled frequent recapturing of plans.

Reply:
FYI I went with enum even though we don't collect "no plan" plan type